### PR TITLE
Add setup_tb_logging example to getting started tutorial

### PR DESCRIPTION
**Pull request by zeel2104**
Summary
- add `setup_tb_logging` import to the Getting Started tutorial
- include a minimal helper-based TensorBoard logging example as an alternative to manual logger attachment
- explain that the helper automatically attaches common logging events such as training metrics, learning rate, and evaluator metrics
- mirror the helper-based option in the complete code section as a commented alternative

Notes
- this updates the source notebook in `pytorch-ignite/examples`:
  `tutorials/beginner/01-getting-started.ipynb`
- the docs site markdown in `pytorch-ignite.ai` is generated from this notebook, so updating the notebook is the correct source change

### CODE REVIEW

File: tutorials/beginner/01-getting-started.ipynb [hunk 1]
```diff
@@ -106,7 +106,8 @@
     "from ignite.engine import Engine, Events, create_supervised_trainer, create_supervised_evaluator\n",
     "from ignite.metrics import Accuracy, Loss\n",
     "from ignite.handlers import ModelCheckpoint\n",
-    "from ignite.contrib.handlers import TensorboardLogger, global_step_from_engine"
+    "from ignite.contrib.handlers import TensorboardLogger, global_step_from_engine\n",
```
Comment (aaishwarymishra) [inline tutorials/beginner/01-getting-started.ipynb]:
```suggestion
    "from ignite.handlers import TensorboardLogger, global_step_from_engine\n",
```

File: tutorials/beginner/01-getting-started.ipynb [hunk 2]
```diff
@@ -657,6 +685,7 @@
     "from ignite.metrics import Accuracy, Loss\n",
     "from ignite.handlers import ModelCheckpoint\n",
     "from ignite.contrib.handlers import TensorboardLogger, global_step_from_engine\n",
```
Comment (aaishwarymishra) [inline tutorials/beginner/01-getting-started.ipynb]:
```suggestion
    "from ignite.handlers import TensorboardLogger, global_step_from_engine\n",
```

**Review by aaishwarymishra — CHANGES_REQUESTED** [review]

**Comment by vfdev-5** [comment]
@zeel2104 thanks for the PR, can you please address the comments of @aaishwarymishra, thanks!

**Comment by zeel2104** [comment]
@vfdev-5 
Updated both occurrences to import `TensorboardLogger` and `global_step_from_engine` from `ignite.handlers`. Thanks!

**Review by aaishwarymishra — APPROVED** [review]

**Comment by aaishwarymishra** [comment]
@vfdev-5 I think this can be merged.
